src/cryptoadvance/specter/server_endpoints/auth.py

```diff
@@ -141,9 +146,13 @@ def register():
                 "explorers": {"main": "", "test": "", "regtest": "", "signet": ""},
                 "hwi_bridge_url": "/hwi/api/",
             }
+
+            # TODO: These four lines should be a single call to UserManager
```

Comment:
Yes, i agree! Why not do it now?

Reply:
I was trying to minimize the footprint of the PR, but yes, will update with this change!